### PR TITLE
[#1321] add logger request server serviceName/ip/port

### PR DESCRIPTION
**Pull request by chengyouling**

### CODE REVIEW

File: spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/configration/dynamic/GovernanceProperties.java [hunk 1]
```diff
@@ -30,6 +30,8 @@ public class GovernanceProperties {
 
   public static final int WEB_FILTER_SERVICE_AUTH_ORDER = -60000;
 
+  public static final int WEB_FILTER_REQUEST_SERVICE_LOGGER_ORDER = -35000;
```
Comment (liubao68) [inline spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/configration/dynamic/GovernanceProperties.java:33]: WEB_FILTER_REQUEST_LOGGER_ORDER

File: spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/configration/dynamic/GovernanceProperties.java [hunk 2]
```diff
@@ -56,6 +58,9 @@ public class GovernanceProperties {
   public static final String WEBCLIENT_FAULT_INJECTION_ENABLED =
       PREFIX + "." + "webclient.faultInjection.enabled";
 
+  public static final String WEBCLIENT_REQUEST_SERVICE_LOGGER_ENABLED =
```
Comment (liubao68) [inline spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/configration/dynamic/GovernanceProperties.java:61]: WEBCLIENT_REQUEST_LOGGER_ENABLED

File: spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/configration/dynamic/GovernanceProperties.java [hunk 3]
```diff
@@ -56,6 +58,9 @@ public class GovernanceProperties {
...
+      PREFIX + "." + "webclient.requestServiceLogger.enabled";
```
Comment (liubao68) [inline spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/configration/dynamic/GovernanceProperties.java:62]: requestServiceLogger -> requestLogger

File: spring-cloud-huawei-governance/src/main/java/com/huaweicloud/governance/adapters/RequestServiceInfoLoggerUtil.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+import com.huaweicloud.common.context.InvocationContext;
+import com.huaweicloud.governance.GovernanceConst;
+
+public class RequestServiceInfoLoggerUtil {
+  private static final Logger LOGGER = LoggerFactory.getLogger(RequestServiceInfoLoggerUtil.class);
+
+  public static void logServiceInfo(InvocationContext context, Throwable e) {
+    if (context != null && context.getLocalContext(GovernanceConst.CONTEXT_CURRENT_INSTANCE) != null) {
+      if (context.getLocalContext(GovernanceConst.CONTEXT_CURRENT_INSTANCE) instanceof ServiceInstance) {
```
Comment (liubao68) [inline spring-cloud-huawei-governance/src/main/java/com/huaweicloud/governance/adapters/RequestServiceInfoLoggerUtil.java:32]: CONTEXT_CURRENT_INSTANCE 还可能是其他类型吗？

File: spring-cloud-huawei-governance/src/main/java/com/huaweicloud/governance/adapters/webclient/ServiceInfoLoggerExchangeFilterFunction.java [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+public class ServiceInfoLoggerExchangeFilterFunction implements ExchangeFilterFunction, Ordered {
+  private static final Logger LOGGER = LoggerFactory.getLogger(ServiceInfoLoggerExchangeFilterFunction.class);
+
+  @Override
+  public Mono<ClientResponse> filter(ClientRequest request, ExchangeFunction next) {
+    return next.exchange(request).map(response -> logServiceInfo(response, request));
+  }
+
+  private ClientResponse logServiceInfo(ClientResponse response, ClientRequest request) {
+    if (response.statusCode() != HttpStatusCode.valueOf(200)) {
```
Comment (liubao68) [inline spring-cloud-huawei-governance/src/main/java/com/huaweicloud/governance/adapters/webclient/ServiceInfoLoggerExchangeFilterFunction.java:44]: response.statusCode().value() != 200 or family.isSucceful

File: spring-cloud-huawei-governance/src/main/java/com/huaweicloud/governance/adapters/RequestServiceInfoLoggerUtil.java (hunk 4 above)
Comment (chengyouling) [reply spring-cloud-huawei-governance/src/main/java/com/huaweicloud/governance/adapters/RequestServiceInfoLoggerUtil.java:32]: 没有

File: spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/configration/dynamic/GovernanceProperties.java (hunk 1 above)
Comment (chengyouling) [reply spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/configration/dynamic/GovernanceProperties.java]: fixed

File: spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/configration/dynamic/GovernanceProperties.java (hunk 2 above)
Comment (chengyouling) [reply spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/configration/dynamic/GovernanceProperties.java]: fixed

File: spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/configration/dynamic/GovernanceProperties.java (hunk 3 above)
Comment (chengyouling) [reply spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/configration/dynamic/GovernanceProperties.java]: fixed

File: spring-cloud-huawei-governance/src/main/java/com/huaweicloud/governance/adapters/webclient/ServiceInfoLoggerExchangeFilterFunction.java (hunk 5 above)
Comment (chengyouling) [reply spring-cloud-huawei-governance/src/main/java/com/huaweicloud/governance/adapters/webclient/ServiceInfoLoggerExchangeFilterFunction.java]: fixed

**Comment by qigaoneng** [comment]
OK

**Pull request merged**